WordPress/Classes/ViewRelated/Tools/SettingsMultiTextViewController.m

```diff
@@ -160,4 +160,26 @@ - (void)adjustCellSize
     [self.tableView endUpdates];
 }
 
+#pragma mark - UITextViewDelegate
+
+- (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range replacementText:(NSString *)text {
+    
+    // If character length is unrestricted, allow all text.
+    if (self.maxCharacterCount <= 0) {
+        return YES;
+    }
+    
+    NSString *newText = [textView.text stringByReplacingCharactersInRange: range withString: text];
+    
+    // If the entire new text will fit, allow it.
+    if (newText.length <= self.maxCharacterCount)
+    {
```

Comment:
nit
```suggestion
    if (newText.length <= self.maxCharacterCount) {
```

Reply:
Thanks @mindgraffiti . I had to fix another one just I just did another commit.